hrms/hr/doctype/attendance_request/attendance_request.json

```diff
@@ -12,6 +12,7 @@
   "employee_name",
   "department",
   "company",
+  "ignore_holiday",
```

Comment:
As a user, this is confusing for me. "Ignore holiday" can be perceived as ignoring holidays in between dates while marking attendance which the system is already doing.
Should be "Mark attendance for holidays"/"Include Holidays" instead IMO. Or any suggestion for a better label?

Also, this field should be placed below the date fields (after the half day date field)